src/Comment.jsx

```diff
@@ -6,33 +6,47 @@ import { RIETextArea } from 'riek';
 import PropTypes from 'prop-types';
 import '../style/index.css';
 import TimeAgo from 'react-timeago';
+import CommentClient from './CommentClient';
 
-export default class Comment extends React.Component {
+if (!global.globalNameCache) {
+  global.globalNameCache = {};
+}
+
+export default class _Comment extends React.Component {
 
   constructor (props) {
     super(props);
+    this.commentClient = new CommentClient(props.accessToken, props.commentUri);
     this.state = {
-      comment: '',
-      createdBy: null,
-      createdByName: null,
```

Comment:
I notice that some default values are `''` and others are `null`. Is there a reason?

Reply:
Some components I am using require not null content. So the default is empty string. The comment is always displayed regardless of what it is.

In case of the others, there is actually logic what to display. If createByName is null, the component doesn't attempt to display it in any way, but instead tries to display the id, and if that is missing displays nothing.